src/clients/index.js

```diff
@@ -0,0 +1,25 @@
+import fs from 'fs';
+
+const handlers = _generateHandlers();
+
+export {
+  getHandler
+};
+
+function getHandler(messageType) {
```

Comment:
Instead of having the `exports {...}` and then somewhere else the actual code just use:

```js
export function getHandler(messageType) {
  // stuff
}
```